### DISCUSSION TITLE
does "ENABLE_NATIVE_OPTIMIZATION" works ?

### DISCUSSION THREAD

**Discussion by Satinelamp** [deepmodeling/abacus-develop#3007] [Q&A]
I noticed that there is a building option called `ENABLE_NATIVE_OPTIMIZATION`, does this option accelerate abacus?
I tried to use this option, doesn't get a better performance.

**Comment by caic99** [comment]
Hi @Satinelamp ,
This option will make the compiler use of all the available instructions of CPU. From my test result it would make ABACUS run ~10% faster.
However, the heavy-lifting calculations is accomplished in third-party libraries, like ELPA and OpenBLAS. I would suggest you using corresponding flags on them to gain a better performance.
